app/src/components/filterEntities/containers/filterEntitiesURLContainer.jsx

```diff
@@ -48,22 +48,78 @@ const FilterEntitiesURL = ({
     debounceTime,
   ]);
 
-  return render({
+  return {
     entities,
     onChange: debounced ? debouncedHandleChange : handleChange,
-  });
+  };
 };
 
 FilterEntitiesURL.propTypes = {
   entities: PropTypes.object,
   updateFilters: PropTypes.func,
-  render: PropTypes.func.isRequired,
   debounced: PropTypes.bool,
   debounceTime: PropTypes.number,
   defaultPagination: PropTypes.any.isRequired,
   prefixQueryKey: PropTypes.string,
 };
 
+const filterEntitiesURLForContainer = ({
+  entities = {},
+  updateFilters = () => {},
+  debounced = true,
+  debounceTime = 1000,
+  defaultPagination,
+  prefixQueryKey,
+  render,
+}) => {
+  const filterEntitiesURLParams = FilterEntitiesURL({
+    entities,
+    updateFilters,
+    debounced,
+    debounceTime,
+    defaultPagination,
+    prefixQueryKey,
+  });
+  return render(filterEntitiesURLParams);
+};
+
+export const withFilterEntitiesURL = (namespace, prefixQueryKey) => (WrappedComponent) => {
```

Comment:
What is the reason to create almost the same logic as for `FilterEntitiesURL`?
If the main reason is a `namespace`, it can be safely added as a second parameter to the `createFilterEntitiesURLContainer` and used when needed.

Reply:
@BlazarQSO ^

Reply:
I can't simplify it